### PR TITLE
Whitelist of Prometheus cAdvisor metrics

### PR DESCRIPTION
**Pull request by mvladev**
Following cAdvisor metrics are now whitelisted:

- container_last_seen
- container_fs_usage_bytes
- container_fs_inodes_total
- container_fs_limit_bytes
- container_network_receive_bytes_total
- container_network_transmit_bytes_total
- container_cpu_cfs_periods_total
- container_cpu_usage_seconds_total
- container_memory_usage_bytes
- container_memory_working_set_bytes

Other metrics will be drop when ingested by Prometheus.
Bumped version of Prometheus to 2.3.2

**What this PR does / why we need it**: Greatly reduces memory usage in Prometheus by dropping unused metrics

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Prometheus collects less metrics from cAdvisor
```

### CODE REVIEW

File: charts/seed-monitoring/charts/prometheus/templates/config.yaml [hunk 1]
```diff
@@ -179,17 +176,23 @@ data:
...
+        regex: ^container_({{ .Values.cadvisorAllowedMetrics | join "|" }})$
+        action: keep
+      - source_labels: [ container_name ]
+        regex: ^POD$
         action: drop
-      - source_labels: [ __name__ ]
-        regex: container_cpu_load_average_10s
+        # drop terraform pods
+      - source_labels: [ pod_name ]
+        regex: ^.+\.tf-pod.+$
```
Comment (dkistner) [inline charts/seed-monitoring/charts/prometheus/templates/config.yaml]: Why we have to drop terraform pod metrics in the Shoot cluster? This is the scrape job of the `cadvisor` for the kubelets of the Shoot and not for the kubelets of the Seed, right?

**Review by dkistner — COMMENTED** [review]
Seems good just a small remark

File: charts/seed-monitoring/charts/prometheus/templates/config.yaml (hunk 1 above)
Comment (mvladev) [reply charts/seed-monitoring/charts/prometheus/templates/config.yaml]: Prometheus can't reliably scrape those metrics - in between scrape intervals the Terraform pod might start, do some work and stop without leaving a trace. 

Ideally those jobs are going to use [PushGateway](https://github.com/prometheus/pushgateway) to deliver metrics, but we can't do anything for container job metrics at the moment.

Comment (dkistner) [reply charts/seed-monitoring/charts/prometheus/templates/config.yaml]: Yes, that makes sense, but this is the job to scrape the `cadvisor` of the kubelets, which are running on the **Shoot** nodes, right? In the Shoot will not run any Gardener terraform pods, therefore there will be no such metrics to drop? Am I right?

**Comment by mvladev** [comment]
removed the dropping of terraform pods for the shoot's Prometheus

**Review by dkistner — APPROVED** [review]
lgtm.

**Review by rfranzke — APPROVED** [review]

**Comment by rfranzke** [comment]
@mvladev still consuming way too much memory:
```
prometheus-0                              2/2       Running     134        18h
```

```
...
  containerStatuses:
  - containerID: docker://6ab28008c28035599615d8f18fd7d847407574066f69ee7ad20f3e2fca23db9c
    image: prom/prometheus:latest
    imageID: docker-pullable://prom/prometheus@sha256:7d99455a045a6c89c0dbee6e1fe659eb83bd3a19e171606bc0fd10eb0e34a7dc
    lastState:
      terminated:
        containerID: docker://18c48166e06996ddf594574e39fefb0427c92e448194425149be0d403a64c622
        exitCode: 137
        finishedAt: 2018-08-01T08:32:31Z
        reason: OOMKilled
        startedAt: 2018-08-01T08:25:35Z
    name: prometheus
    ready: false
    restartCount: 134
    state:
      running:
        startedAt: 2018-08-01T08:33:56Z
```

**Pull request merged**